modules/olm-policy-scoping-operator-install.adoc

```diff
@@ -41,9 +50,39 @@ metadata:
   namespace: scoped
 EOF
 ----
+====
+
+.. Create a secret by running the following command:
++
+.Example creation of a long-lived API token `Secret` object
+[%collapsible]
+====
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/service-account-token <1>
+metadata:
+  name: scoped
+  namespace: scoped
+  annotations:
+    kubernetes.io/service-account.name: scoped
+EOF
+----
+<1> The secret must be a long-lived API token, which is used by the service account.
+====
+
+.. Create a role by running the following command.
 +
-The following example grants the service account permissions to do anything in the designated namespace for simplicity. In a production environment, you should create a more fine-grained set of permissions:
+[WARNING]
+====
+In this example, the role grants the service account permissions to do anything in the designated namespace for simplicity. In a production environment, you should create a more fine-grained set of permissions. For more information, see "Fine-grained permissions".
```

Comment:
I know this is existing text, but it isn't immediately clear what _for simplicity_ refers to.

 ```suggestion
In this example, the role grants the service account permissions to do anything in the designated namespace for demonstration purposes only. In a production environment, you should create a more fine-grained set of permissions. For more information, see "Fine-grained permissions".
```